WordPress/gutenberg
# Env: Remove version field from docker-compose config

## What?
Remove the `version` field that's written to the `docker-compose.config.yml` file generated by the env package.
Fixes https://github.com/WordPress/gutenberg/issues/63073.

## Why?
The field is obsolete and has started causing automated tests to fail (due to a warning about it being obsolete), see e.g https://github.com/WordPress/gutenberg/actions/runs/9776598064/job/26989677669?pr=62092.

We have some precedent for this in `wordpress-develop`, see [`#61101`](https://core.trac.wordpress.org/ticket/61101) and https://github.com/WordPress/wordpress-develop/pull/6492.

## How?
By removing that field 🤷‍♂️ 

## Testing Instructions
Verify that tests pass in CI. Alternatively, try locally. You might want to destroy your current wp-env setup and recreate it.
